tests/integration/conftest.py

```diff
@@ -0,0 +1,374 @@
+import os
+from dataclasses import dataclass
+from datetime import datetime
+import decimal
+import random
+
+import pytest
+from pyspark.sql import SparkSession
+from pyspark.sql.types import (
+    StructType,
+    StructField,
+    StringType,
+    IntegerType,
+    LongType,
+    DecimalType,
+    DateType,
+    DoubleType,
+    Row,
+)
+from databricks.connect import DatabricksSession
+from databricks.sdk import WorkspaceClient
+from databricks.sdk.service.catalog import VolumeType
+
+from databricks.labs.remorph.config import ReconcileConfig, DatabaseConfig, ReconcileMetadataConfig, MorphConfig
+from databricks.labs.remorph.helpers import db_sql
+from databricks.labs.remorph.helpers.deployment import TableDeployer
+from databricks.labs.remorph.install import ReconciliationMetadataSetup, CatalogSetup
+
+
+@dataclass
+class TestConfig:
+    db_table_catalog: str
+    db_table_schema: str
+    db_table_name: str
+    db_mock_src: str
+    db_mock_tgt: str
+
+
+@dataclass
+class TestMetricsConfig:
+    catalog: str
+    schema: str
+    volume: str
+
+
+def _create_ws() -> WorkspaceClient:
+    # Use variables from Unified Auth
+    # See https://databricks-sdk-py.readthedocs.io/en/latest/authentication.html
+    product_name, product_version = None, None
+    return WorkspaceClient(host=os.environ["DATABRICKS_HOST"], product=product_name, product_version=product_version)
+
+
+def _create_spark_session(ws: WorkspaceClient) -> SparkSession:
+    return DatabricksSession.builder.sdkConfig(ws.config).getOrCreate()
+
+
+def _get_metrics_config() -> TestMetricsConfig:
+    return TestMetricsConfig(catalog="integration_test", schema="reconcile", volume="test_volume")
+
+
+def _get_test_config() -> TestConfig:
+    return TestConfig(
+        db_table_catalog="samples",
```

Comment:
keep in mind that all catalogs are wiped out by our test infrastructure, so you have to pre-create randomly-named catalogs for every tests and tear them down. actually... let's create a test catalog just for you

Reply:
tear down process is already in place to cleanup the catalog. Let me know if I missed out anything or any alternatives on that.